test/mobile/model_test/gen_test_model.py

```diff
@@ -89,7 +89,8 @@
     "nn_utils_ops": NNUtilsModule(),
     # quantization ops
     "general_quant_ops": GeneralQuantModule(),
-    "dynamic_quant_ops": DynamicQuantModule(),
+    # TODO(sdym@fb.com): fix and re-enable dynamic_quant_ops
+    # "dynamic_quant_ops": DynamicQuantModule(),
```

Comment:
Could we have an issue to track this?

Reply:
> Could we have an issue to track this?

Done, https://github.com/pytorch/pytorch/issues/82139